view/app-jarvis.js

```diff
@@ -114,61 +112,80 @@ app.controller('MainController', function($scope,$location,$rootScope,$http) {
 
 	$scope.initStack = function() {
 		$scope.message = 'Type a message ...';
-		$scope.toggleStartStop();
+		$scope.toggleStartStop(0);
 	};
 
-	$scope.toggleStartStop = function() {
+	$scope.toggleStartStop = function (check) {
 		recognition.continuous = true;
 
 		recognition.onresult = function (event) {
-			var n, m, submessage, messa;
+			var n, m, submessage, text;
 			var mess = document.getElementById('message-input');
 			mess.value = '';
-			messa = '';
-			for (var i = 0; i < event.results.length; i++) {
-				if (event.results[i].isFinal) {
-					messa += event.results[i][0].transcript;
-					if (messa.includes('start Jarvis')) {
-						m = messa.lastIndexOf('start Jarvis');
-						submessage = messa.substring(m+12);
-						mess.value = submessage;
-						$scope.message = submessage; 
+			text = '';
+			if (check == 0){
```

Comment:
use === and () {}

Reply:
this is still not changed!

Reply:
I had not pushed it. Please check it now.